server/src/services/initialization/onInitialize.ts

```diff
@@ -38,7 +38,7 @@ export const onInitialize = (serverState: ServerState) => {
         textDocumentSync: TextDocumentSyncKind.Incremental,
         // Tell the client that this server supports code completion.
         completionProvider: {
-          triggerCharacters: [".", "/", '"'],
+          triggerCharacters: [".", "/", '"', "'"],
```

Comment:
Can you walk me through the cases this supports?

Reply:
Is this to support `import './Auth.sol';` for import statements?

Reply:
So I noticed that I wasn't getting import completion, and it was because I use single quotes instead of double, because it's a single keystroke instead of a chord. Even though the formatter may change it to double quotes afterwards, I'll always use single quotes, and thought that this may be happening to a lot of people.

Reply:
Based on the solidity grammar, it looks as if import paths are just `non-empty-string-literals` and so can be either single or double quoted, and that has just been missed previously.

If we are going to fix that as part of this PR we should probably update the completion guard here: https://github.com/NomicFoundation/hardhat-vscode/blob/f8d9498a4f386484218647eb18b86c0aa99e7158/server/src/services/completion/onCompletion.ts#L138